File: app/src/main/java/com/example/whattowatch/model/mappers/VideoMapper.java
package com.example.whattowatch.model.mappers;

import com.example.whattowatch.model.mymodel.MyVideoModel;
import com.example.whattowatch.model.response.video.MovieVideos;

import java.util.ArrayList;
import java.util.List;

public class VideoMapper {
    public static MyVideoModel convertToMyVideoModel(MovieVideos movieVideos){

        return new MyVideoModel(movieVideos.getName(),movieVideos.getKey());
    }

    public static List<MyVideoModel> convertToMyVideoList(List<MovieVideos> movieVideos){
        List<MyVideoModel> videoModelList = new ArrayList<>();

        for(MovieVideos mv: movieVideos){
            videoModelList.add(convertToMyVideoModel(mv));
        }
        return videoModelList;
    }
}

File: app/src/main/java/com/example/whattowatch/model/mappers/VideoMapper.kt
package com.example.whattowatch.model.mappers

import com.example.whattowatch.model.mymodel.MyVideoModel
import com.example.whattowatch.model.response.video.MovieVideos
import java.util.*

object VideoMapper {
    fun convertToMyVideoModel(movieVideos: MovieVideos): MyVideoModel {
        return MyVideoModel(movieVideos.name, movieVideos.key)
    }

    fun convertToMyVideoList(movieVideos: List<MovieVideos>): List<MyVideoModel> {
        val videoModelList: MutableList<MyVideoModel> = ArrayList()
        for (mv in movieVideos) {
            videoModelList.add(convertToMyVideoModel(mv))
        }
        return videoModelList
    }
}